Project/src/main/java/app/ReviewerController.java

```diff
@@ -21,35 +33,53 @@
      */
     @GetMapping("/reviewer")
     public String getReviewer(Model model) {
-        model.addAttribute("articles", articleRepository.findByUsersOrderByDateAsc(super.getUser()));
+        model.addAttribute("articles", articleRepository.findByUsersOrderByReviewDueDateAsc(super.getUser()));
         return "reviewer";
     }
 
     /***
-     * Gets the information about a single article
-     * @param model The model to return the files
-     * @param id The id of the article
-     * @return The review view
+     * Uploads a file to a review
+     * @param file The Multipart file that got uploaded
+     * @return
+     * @throws IOException
      */
-    @GetMapping("/review/{id}")
-    public String getReviewer(Model model, @PathVariable String id) {
-        model.addAttribute("article", articleRepository.findById(Long.parseLong(id)).get());
-        return "review";
+    @PostMapping("/review")
+    public String uploadFile(@RequestParam String articleId, @RequestParam("file") MultipartFile file) throws IOException {
+        if (file == null || StringUtils.isNullOrEmpty(file.getOriginalFilename())
+                || StringUtils.isNullOrEmpty(file.getContentType()) || file.getBytes().length <= 0) {
+            //This is how spring handles bad requests, throws exceptions.
+            throw new BadFileException();
+        }
+        articleRepository.findById(Long.parseLong(articleId)).ifPresent((Article article)  -> {
```

Comment:
Instead of doing Long.parseLong you can just change the type of parameter to be a Long.